SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DAQmx/Types.cs

```diff
@@ -109,5 +117,13 @@ public class DAQmxTimingSampleClockSettings
         /// This value is used to get/set the value of the NationalInstruments.DAQmx.Timing.SampleClockSource driver property.
         /// </remarks>
         public int? SamplesPerChannel { get; set; }
+
+        /// <summary>
+        /// Creates an object to define and set DAQmx Timing Sample Clock specific settings.
+        /// This object is used as a parameter for configuring a timing sample clock for DAQmx task <see cref="Configure.ConfigureTiming(DAQmxTasksBundle, DAQmxTimingSampleClockSettings)"/>
```

Comment:
... for configuring sample clock timing type for a DAQmx task.

Reply:
Updated